### DISCUSSION TITLE
Example of a user interface for manual data upload ?

### DISCUSSION THREAD

**Discussion by fpavogt** [wmo-im/wis2box#774] [Q&A] [answered]
Dear hivemind,

Is there somehwere an exemple of a WIS 2.0 box [data ingestion user interface](https://docs.wis2box.wis.wmo.int/en/1.0b8/user/data-ingest.html) that I could access ? I would like to see how it looks and behaves overall, and (in particular) how well it works (and how comfortable it is) on mobile devices. 

Has anyone some experience on the front of manual data ingestion ? I'm trying to figure what sort of minimal equipment is required for an observer in the field to submit an hourly measurement report, and whether additional develpoments might be needed to make the default setup truly "mobile friendly" ?

**Comment by maaikelimper** [comment, marked as answer]
Hi,
You can see a screenshot of the form for manual observers here:
![image](https://github.com/user-attachments/assets/0a4db919-c333-464c-8295-3110a800fcd5)

See also this section in the training website:
https://training.wis2box.wis.wmo.int/practical-sessions/converting-synop-data-to-bufr-form/

If you want to see how well it works on mobile devices I recommend spinning up a wis2box instance yourself, using the instructions provided at:
https://docs.wis2box.wis.wmo.int/en/latest/user/getting-started.html

I am not sure the default setup is mobile friendly.

A developer can relatively easy develop mobile friendly version for uploading data, by making a http POST request to the process included in the wis2box-api, `/oapi/processes/wis2box-synop2bufr` or `/oapi/processes/wis2box-csv2bufr`

**Reply by fpavogt** [reply]
Thank you for the infos and screenshot - much appreciated and very useful !
